README.md

```diff
@@ -41,28 +41,60 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
 ### Linux
 
 1. Visit [release page](https://github.com/iNavFlight/inav-configurator/releases)
-1. Download Configurator for Linux platform (linux32 and linux64 are present)
-1. Extract tar.gz archive
-1. Make the following files executable:
-   * inav-configurator `chmod +x inav-configurator`
-   * (5.0.0+) chrome_crashpad_handler `chmod +x chrome_crashpad_handler`
-1. Run INAV Configurator app from unpacked folder
+2. Download Configurator for Linux platform (linux32 and linux64 are present)
+   *  **.rpm** is the Fedora installation file. Just download and install using `sudo dnf localinstall /path/to/INAV-Configurator_linuxNN_x.y.z..rpm` or open it with a package manager (e.g. via Files)
```

Comment:
Looks like we have 1 . too many in the rpm file name

Reply:
The filename wasn't quite right either. Corrected it and the .deb.